Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/exposureriskmetadata/ExposureRiskMetadataDonor.kt

```diff
@@ -74,7 +75,7 @@ class ExposureRiskMetadataDonor @Inject constructor(
         val riskLevelPtForMetadata = lastPTRiskResult.toMetadataRiskLevel()
         val mostRecentDateAtPtRiskLevel = lastPTRiskResult
             .mostRecentDateAtRiskState
-            ?.toDateTimeAtStartOfDay()
+            ?.toDateTimeAtStartOfDay(DateTimeZone.UTC)
             ?.toInstant()
```

Comment:
Can you add a unit test for that?

Reply:
Thanks for the test. Can you add one for the donor to make sure this is and stays fixed?

Reply:
Changed present tests to use toDateTimeAtStartOfDayUtc